src/mpyl/cli/repository.py

```diff
@@ -160,10 +162,12 @@ def init(obj: RepoContext, url: str, pull: int, branch: str, pristine: bool):
 
     console.log("Preparing repository for a new run...")
 
-    with Repository.from_shallow_diff_clone(
-        branch.replace("refs/heads/", ""), url, "main", obj.config, Path(".")
-    ) if pristine else Repository(
-        config=RepoConfig.from_config(parse_config(obj.config))
+    with (
```

Comment:
I've seen these changes more often in your pr's, just wondering: is this Black or your IDE settings?

Reply:
I'm not sure. I have IntelliJ configured to run Black on save:
![image](https://github.com/Vandebron/mpyl/assets/516602/b7b56860-cc7e-49d1-8838-1bc7307de941)
With (what I think are) the default settings:
![image](https://github.com/Vandebron/mpyl/assets/516602/06e20727-3c0e-41f9-8e0f-ea98297e64df)

Not sure why it ends up with different styling 🤷